core/src/main/java/com/rudderstack/android/sdk/core/DBPersistentManager.java

```diff
@@ -466,11 +455,12 @@ private void deleteStatusColumn(SQLiteDatabase database) {
 
     private boolean checkIfStatusColumnExists(SQLiteDatabase database) {
         String checkIfStatusExistsSqlString = "PRAGMA table_info(events)";
+        Cursor allRows = null;
         try {
```

Comment:
I think we can do try with resources for closeables. That's recommended approach

Reply:
yes, we can do that, will change it accordingly

Reply:
addressed it